CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Bug Fixes
 
 - [#1058](https://github.com/crypto-org-chain/cronos/pull/1058) Fix decode log for multi topics in websocket subscribe ([ethermint commit](https://github.com/crypto-org-chain/ethermint/commit/2136ad029860c819942ad1836dd3f42585002233)).
+- [#1062](https://github.com/crypto-org-chain/cronos/pull/1062) Update cometbft `v0.34.29` with several minor bug fixes and low-severity security-fixes.
```

Comment:
not sure if we release with v1.0.10 together

Reply:
no problem